packages/terra-time-input/translations/es-ES.json

```diff
@@ -4,5 +4,7 @@
   "Terra.timeInput.hours": "Horas",
   "Terra.timeInput.minutes": "Minutos",
   "Terra.timeInput.hh": "HH",
-  "Terra.timeInput.mm": "mm"
+  "Terra.timeInput.mm": "mm",
+  "Terra.timeInput.display.meridiem": "",
+  "Terra.timeInput.select.meridiem": ""
```

Comment:
Should es-ES and es translations match es-US translations?

Reply:
no, translation team noted that es-US should use 12 hour format, but es should use 24 hour format.

Reply:
@bjankord Interesting! Makes sense then 😄